docs/integrations/python.md

```diff
@@ -147,7 +147,7 @@ print(repo)
 import lakefs
 
 print("Listing repositories:")
-for repo in lakefs.repositories():
+for repo in lakefs.repositories(client=clt):
```

Comment:
This is not the main use-case. The main use case is to use the default client.
I assuming this is relevant for the SSL configuration